public/javascripts/ng-controllers.js

```diff
@@ -108,3 +109,44 @@ springInitiative.controller('mainController', function($scope, $http) {
   };
 
 });
+
+springInitiative.controller('indexController', function($scope, $rootScope, $http, $location){
+    $scope.students = [];
+
+    // checkUser($rootScope, $location, $http);
+    $scope.user = $rootScope.loggedInUser;
+    console.log($rootScope.loggedInUser);
+
+    // TODO: make this cleaner using http.get(...)
+    $http({
+          method: 'GET',
+          url: '/api/allStudents'
+        })
+        .success(function(data){
+            console.log(data);
+            $scope.students = data;
+         })
+        .error(function(err){
+             console.log('Error: in GET \'/student\'', err);
+        }
+    );
+});
+
+springInitiative.controller('overviewController', function($scope, $rootScope, $http, $location){
+});
+
+springInitiative.controller('programController', function($scope, $rootScope, $http, $location){
+    $scope.programName = 'something spring';
+    $scope.programInfo = ':3';
+});
+
+springInitiative.controller('studentController', function($scope, $rootScope, $http, $location){
+    $scope.studentName = 'd';
+    $scope.studentInfo = 'we';
+});
+
+springInitiative.controller('addStudentController', function($scope, $rootScope, $http, $location){
+});
+
+springInitiative.controller('settingsController', function($scope, $rootScope, $http, $location){
+});
```

Comment:
@aconsilvio the controller code you wrote is starting at line 75 in the `ng-controllers` file.